src/main/java/uk/gov/hmcts/reform/sscs/service/ServiceHearingsService.java

```diff
@@ -48,10 +45,13 @@ public ServiceHearingValues getServiceHearingValues(ServiceHearingRequest reques
         String originalCaseData = objectMapper.writeValueAsString(caseData);
 
         ServiceHearingValues model = ServiceHearingValuesMapping.mapServiceHearingValues(caseData, refData);
+        boolean hasInValidHearingVideoEmail = getHasInValidHearingVideoEmail(caseData);
 
         String updatedCaseData = objectMapper.writeValueAsString(caseData);
 
-        if (!originalCaseData.equals(updatedCaseData)) {
+        if (hasInValidHearingVideoEmail) {
+            updateCaseDataToListingError(caseData, "Hearing video email address must be valid email address");
```

Comment:
This needs to be moved to the hearingsService and a listingError needs to be thrown and caught and then handled in the tribs queue class